src/workerd/server/workerd.capnp

```diff
@@ -651,8 +662,17 @@ struct Worker {
   streamingTails @15 :List(ServiceDesignator);
   # List of streaming tail worker services that should receive tail events for this worker.
   # NOTE: This will be deleted in a future refactor, do not depend on this.
+
+  containerService @16 :ServiceDesignator;
```

Comment:
Do we need an external service in C++ - I think that's what Kenton mentioned, this could be local.